plugins/woocommerce/includes/class-wc-order.php

```diff
@@ -6,6 +6,8 @@
  * @version 2.2.0
  */
 
+use Automattic\WooCommerce\Checkout\Enums\OrderStatus;
```

Comment:
(2/2) Also following https://github.com/woocommerce/woocommerce/pull/51919/commits/72ef37d4a0002ae764cb0ea1c97b870f3dc05b70, we should make a further change here:

```suggestion
use Automattic\WooCommerce\Enums\OrderStatus;
```